infrastructures/kubernetes/src/test/java/org/eclipse/che/workspace/infrastructure/kubernetes/KubernetesInternalRuntimeTest.java

```diff
@@ -410,6 +410,7 @@ public void startsKubernetesEnvironment() throws Exception {
     verify(services).create(any());
     verify(secrets).create(any());
     verify(configMaps).create(any());
+    verify(namespace).cleanUp();
```

Comment:
I know, currently, the test does not respect the order but still, it would be better to put this call where it's expected to be called, somewhere before `verify(deployments).deploy(any(Pod.class))`

Reply:
I've added an extra test for this case, to not increase complexity of this one...